src/pages/RestaurantPageInfo/RestaurantPageInfo.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PlaceDescription from '../../components/PlaceDescription/PlaceDescription';
+import OpeningHours from '../../components/OpeningHours/OpeningHours';
+import RestaurantPicture from '../../assets/images/restaurant.jpg';
+import './RestaurantPageInfo.scss';
+import BackButton from '../../components/BackButton/BackButton';
+
+export default function RestaurantInfoPage() {
+  return (
+    <div className="RestaurantPageInfo">
```

Comment:
can we use lowercase, hyphen-separated class here also?

Reply:
Oh, that is an effect of writing code at night or in the late evening. Thank You :smile: